RoddyCore/src/de/dkfz/roddy/execution/jobs/Job.groovy

```diff
@@ -60,6 +60,12 @@ class Job extends BEJob<BEJob, BEJobResult> {
      */
     protected final Map<String, Object> allRawInputParameters
 
+    /**
+     * Keeps a list of all parameters after conversion and before removing them from the final parameter list
+     * (See keepOnlyEssentialParameters)
+     */
+    final Map<String, String> allFinalParameters = [:]
```

Comment:
What other purpose does this variable have, besides providing a parameter list for presentation to the user? Maybe it should be renamed to `reportedParameters`?

`allFinalParameters` is probably even not correct, because one could argue, that even those parameters in the _parameter file are final and should belong to the "all" set.

Reply:
Sound reasonable, will rename it.